File: challenges/6-converter-segundos/ruby/peterjpd/desafio-6.rb
segundos = ARGV[0].to_i

horas = segundos/3600
segundos = segundos%3600
minutos = segundos/60
segundos = segundos%60

horas = format('%02d', horas)
minutos = format('%02d', minutos)
segundos = format('%02d', segundos)

puts "#{horas}:#{minutos}:#{segundos}"

File: challenges/6-converter-segundos/javascript/JoaoSortudo/desafio6.js
let n = process.argv[2]

    let horas = parseInt(n / 3600);
    let minutos = parseInt((n % 3600) / 60);
    let segundos = (n % 3600) % 60;

    if (horas < 10) {
        horas =  `0${horas}`
    }
    if (minutos < 10) {
        minutos = `0${minutos}`
    }
    if (segundos < 10) {
        segundos = `0${segundos}` 
    }

    console.log(horas,":", minutos, ":", segundos);
